llvm/lib/CodeGen/VirtRegMap.cpp

```diff
@@ -474,6 +476,77 @@ bool VirtRegRewriter::readsUndefSubreg(const MachineOperand &MO) const {
   return true;
 }
 
+// Return LaneBitmask value as uint64_t for PhysReg assigned to MO,
+// representing its live register units at its parent MI. In case of undef or
+// fully live MO, return 0u.
+uint64_t VirtRegRewriter::calcLiveRegUnitMask(const MachineOperand &MO,
```

Comment:
Why don't you return LaneBitmask as it is?
This can be later converted to uint64 value using `getAsInteger()` whenever required.